src/generators/bitflag.rs

```diff
@@ -75,13 +75,13 @@ fn bitflag_token_stream(
             }
         }
 
-        impl gio::glib::StaticVariantType for #ident {
+        impl gio::glib::prelude::StaticVariantType for #ident {
```

Comment:
I recommend accessing the traits directly from the `variant` module itself, same with other instances, except on the test.